locations/spiders/amctheatres.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+import json
+
+from locations.items import GeojsonPointItem
+
+
+class AmcTheatresSpider(scrapy.Spider):
+    name = "amctheatres"
+    allowed_domains = ['amctheatres.com']
+    start_urls = (
+        'https://www.amctheatres.com/sitemaps/sitemap-theatres.xml',
+    )
+
+    def parse(self, response):
+        response.selector.remove_namespaces()
+        city_urls = response.xpath('//url/loc/text()').extract()
+        for path in city_urls:
+            yield scrapy.Request(
```

Comment:
Instead of spidering off to the individual theater pages it looks like the sitemap might have everything you're pulling out already:

![image](https://user-images.githubusercontent.com/261584/33811641-7cc68288-dddb-11e7-8abb-327fbc406d6a.png)